src/main/java/org/example/models/JobRole.java

```diff
@@ -0,0 +1,124 @@
+package org.example.models;
+
+import java.util.Date;
+
+public class JobRole {
+    private int roleId;
+    private String roleName;
+    private String description;
+    private String responsibilities;
+    public String locations;
+    private String linkToJobSpec;
+    private String capability;
+    private String band;
+    private Date closingDate;
+    private boolean status;
+    private int positionsAvailable;
+
+    // Removed capability, status, positionsAvailable
+    // from constructor due to 7 parameters limit.
+    public JobRole(
```

Comment:
As you haven't made anything required in the database table, you can't expect the constructor to have all the values

Reply:
This hasn't been completed. Don't ignore comments please :)

Reply:
This still hasn't been done